packages/hub/src/components/EntityCard.tsx

```diff
@@ -17,50 +17,102 @@ function formatDate(date: Date): string {
   return date.toLocaleDateString("en-US", options);
 }
 
+export const entityCardBadgeCss = (presentAsLink: boolean) => {
+  return clsx(
+    "flex items-center",
+    presentAsLink &&
+      "cursor-pointer hover:text-gray-900 hover:underline select-none"
+  );
+};
+
+export const EntityCardBadge: FC<
+  PropsWithChildren<{ presentAsLink: boolean }>
+> = ({ presentAsLink, children }) => (
+  <div className={entityCardBadgeCss(presentAsLink)}>{children}</div>
+);
+
+export const PrivateBadge: FC = () => (
+  <EntityCardBadge presentAsLink={false}>
+    <LockIcon className="mr-1" size={10} />
+    Private
+  </EntityCardBadge>
+);
+
+export const UpdatedStatus: FC<{ time: number }> = ({ time }) => (
+  <EntityCardBadge presentAsLink={false}>
+    <span className="mr-1">{"Updated"}</span>
+    <time dateTime={new Date(time).toISOString()}>
+      {formatDate(new Date(time))}
+    </time>
+  </EntityCardBadge>
+);
+
+export const keepFirstNLines = (str: string, n: number) =>
+  str.split("\n").slice(0, n).join("\n");
+
+const InterspersedMenuItems: React.FC<{
+  items: React.ReactNode[];
```

Comment:
(minor)

If you rename `items` to `children` here (with the same strict type, `ReactNode[]`), it would work in the same way, but you won't have to use keys for individual items, so it would be a bit nicer to use.